components/pagination/README.md

```diff
@@ -0,0 +1,63 @@
+Hods - Pagination
+===
+
+Component to allow separating long tables into separate tables
```

Comment:
Needs a full-stop at the end.

Reply:
Also, is this specific to tables?

I'd say something like:
```
A component for navigating a long list of results.
```

Reply:
updated in latest commit